File: VB/WinWebSolution.Module/Module.vb
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.Xpo.Metadata
Imports DevExpress.Xpo
Imports DevExpress.Persistent.BaseImpl

Namespace WinWebSolution.Module
	Public NotInheritable Partial Class WinWebSolutionModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
		Public Overrides Sub CustomizeXPDictionary(ByVal xpDictionary As XPDictionary)
			MyBase.CustomizeXPDictionary(xpDictionary)
			Dim personClassInfo As XPClassInfo = XafTypesInfo.XpoTypeInfoSource.XPDictionary.GetClassInfo(GetType(Person))
			Dim personFullNameMemberInfo As XPMemberInfo = personClassInfo.GetMember("FullName")
			Dim persistentAliasAttribute As PersistentAliasAttribute = TryCast(personFullNameMemberInfo.FindAttributeInfo(GetType(PersistentAliasAttribute)), PersistentAliasAttribute)
			If persistentAliasAttribute Is Nothing Then
				personFullNameMemberInfo.AddAttribute(New PersistentAliasAttribute("FirstName + MiddleName + LastName"))
			End If
		End Sub
	End Class
End Namespace

File: CS/WinWebSolution.Module/Module.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.Xpo.Metadata;
using DevExpress.Xpo;
using DevExpress.Persistent.BaseImpl;

namespace WinWebSolution.Module {
    public sealed partial class WinWebSolutionModule : ModuleBase {
        public WinWebSolutionModule() {
            InitializeComponent();
        }
        public override void CustomizeXPDictionary(XPDictionary xpDictionary) {
            base.CustomizeXPDictionary(xpDictionary);
            XPClassInfo personClassInfo = XafTypesInfo.XpoTypeInfoSource.XPDictionary.GetClassInfo(typeof(Person));
            XPMemberInfo personFullNameMemberInfo = personClassInfo.GetMember("FullName");
            PersistentAliasAttribute persistentAliasAttribute =
                personFullNameMemberInfo.FindAttributeInfo(typeof(PersistentAliasAttribute)) as PersistentAliasAttribute;
            if (persistentAliasAttribute == null) {
                personFullNameMemberInfo.AddAttribute(new PersistentAliasAttribute("FirstName + MiddleName + LastName"));
            }
        }
    }
}